### ISSUE TITLE
Feature request: Englisch Translation

### ISSUE THREAD

**Issue by gamecompiler** [PingvinOpenTag/LTAscetic#3]
Hi, i and my friends use the LTAscetic project in Germany very much.
Its nice, and works well.
But the problem is, the LCD menu and the LTAscetic Configurator.
Only one in my team speaks the russian language.
Only he can use the Configurator.

This is a feature request(?): Please Translate the menu and the Configurator to English.

Thank you in Advance.

Tipp: I added a current regulator for the LED on the pcb to save power. Works well.

**Comment by PingvinOpenTag** [comment]
Generating SSH keys 
https://help.github.com/articles/generating-ssh-keys/
and send me the key.
I'll add it to the server, and you will have access, if I understand correctly the server.

 >> Your yadi.sk version or release 3_1 ?

Yes.

**Comment by gamecompiler** [comment]
Oh, sorry my handy dont showed me the rest of the e-mail....
Id send it to your github e-mail -> ms_present_god@mail.ru
But i will resend it to your other e-mail.

**Comment by gamecompiler** [comment]
Sorry, I have not started yet, I have huge headaches.
I'll get back to you if I feel good again.

Sorry.

**Comment by gamecompiler** [comment]
Im back, id checked  insensitiveclod´s code on https://github.com/insensitiveclod/LTAscetic.
And i did not found any code difference. 
It took me nearly 7 hours for the review....
The English Translation of his work is almost complete and has a good quality. Please put his code in the main repo if you also think his translation is ok.

**Comment by insensitiveclod** [comment]
On 01/07/2015 02:00 PM, Pingvin wrote:
> OK, I'm put insensiveclod version in main repo.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PingvinOpenTag/LTAscetic/issues/3#issuecomment-69018419>.
> 

Wow:)

Such honor *bow*

The code compiles exactly the same as the version I used before
translation. It's only comments that got translated, prettified and a
bit streamlined.

I will pick up the work of making it 'even nicer' soon, by doing
code-cleanup a bit.
First i will look at the configurator + translation.

-- 
 Arnd

**Comment by gamecompiler** [comment]
Awesome nice. I cant await to hear when you are ready. :)I currently try to find a translation trick for the Firmware which is better then global variables. Maybe we can store Kind of Language files in the EEPROM.(Sorry for those bad Spelling, German T9 in my mobile phone)-------- Originalnachricht --------Betreff: Re: [LTAscetic] Feature request: Englisch Translation (#3)Von: insensitiveclod An: PingvinOpenTag/LTAscetic Cc: Jason On 01/07/2015 02:00 PM, Pingvin wrote:
> OK, I'm put insensiveclod version in main repo.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PingvinOpenTag/LTAscetic/issues/3#issuecomment-69018419>.
> 

Wow:)

Such honor *bow*

The code compiles exactly the same as the version I used before
translation. It's only comments that got translated, prettified and a
bit streamlined.

I will pick up the work of making it 'even nicer' soon, by doing
code-cleanup a bit.
First i will look at the configurator + translation.

-- 
 Arnd

**Comment by PingvinOpenTag** [comment]
I can not add your ssh-key.
I do not know why, but there is an error.
Send it as an archive, try more.

**Comment by gamecompiler** [comment]
Id see you added sth you call Anti-Cheat.
Who does this Anti-Cheat system work ?

Thanks in Advance.

**Comment by PingvinOpenTag** [comment]
Anti-cheat does not allow the player to disconnect bandana during the game on or off Bluetooth bandanna.
That it can not be Duncan Macleod ;-)

**Comment by gamecompiler** [comment]
I cant find the TDA7052 on Mouser as DIP.
We may need another Distributer for our parts.
Or we change the PCB, to let the SMD Version fit.

Or could i use a similiar IC ?
Id found the NJM386 IC.
Its similiar, has more features, we could change the volume over a gain pin.
And it has output power of 1 Watts.

http://www.mouser.com/ds/2/294/NJM386_E-232003.pdf

I would prefer to use mouser because mouser ships worldwide and is very cheap in conclusion to my known distributors.

**Comment by PingvinOpenTag** [comment]
Began to correct the source firmware for multi-language support.
I use this method, in header file 

#define EN 0
#define RU 1

#define LANGUAGE EN

in C file

#if LANGUAGE == RU
lcd_puts("Ключ ТМ");
#elif LANGUAGE == EN
lcd_puts("ТМ key");
#endif
lcd_gotoxy(0, 1);
#if LANGUAGE == RU
lcd_puts("записан!");
#elif LANGUAGE == EN
lcd_puts("recorded");
#endif

**Comment by Teie** [comment]
Hi Pingvin, I'm Theuer from Brazil.
 GREAT JOB you made here!
I'm doing your project at protonboard, I tried open the ".c" files at maaaany softwares because my idea was to do "copy" Rusian COMMENTS and "paste" at Google Translator, but the comments not appears in Cyrillic. All comments appears like this: "//«‡ÔËÒ¸ ·‡ÈÚ‡ ‚ ÏÓ‰ÛÎ¸ Ô‡ÏˇÚË EEPROM"
 Do you know why?
 Others thing, I did not understand what it is for the EEPROM memory. Is it for Score? (Please say yes!) :)

 Sorry for terrible english, and congratulations for your hard work!
 See you, Theuer.

**Comment by gamecompiler** [comment]
The Translation is already (almost) done, please try to read the issues first before you comment.
https://github.com/PingvinOpenTag/LTAscetic/tree/master/internationalized_firmware

Im not sure for what the EEPROM memory is for.
If i remember correct, it contains the playerid and settings like ammo, battery settings (Max and min values when the battery is full or empty e.t.c), health, damage rate....

If im correct there is no score feature, but if you like to add it, please feel free to share your code.

If you have more question, please feel free to ask ;D
Sorry for my english.

**Comment by Teie** [comment]
I read the issues, but I had understood that the partial translation was in "internationalized_firmware" and when I opened, everything I saw was "//«‡ÔËÒ¸ ·‡ÈÚ‡ ‚ ÏÓ‰ÛÎ". Now I opened "intensitivecloud" and I found it all translated.

 Yes, my goal is to write score feature. My current system is MicroMT, and I'm dropping it because they lack the Score feature.
 My big question about ASCETIC was it can handle or not the Score feature, and today Pingvin answer:
 "Yes, of course - it is able to."
 Challenge accepted! :)
